packages/@okta/vuepress-theme-default/components/Sidebar.vue

```diff
@@ -67,6 +92,24 @@
           })
         }
         return false
+      },
+
+      expandSubNav: function (event) {
+        var parent = event.target.parentElement;
```

Comment:
Unless we're dealing with old engines, I recommend against using `var` - prefer `const`, use `let` if we're reassign the variable.  Do we have a specific reason to use `var` here?